code/Belgium/Belgium-2020-03-31.py

```diff
@@ -23,8 +25,10 @@
 day_of_march_ICU  = day_of_march
 ICU =             [ 24,  33,  53,  79, 100, 130,  164,  238,  290,  322,  381,  474,  605,  690,  789,  867,  927, 1021]
 # This included 94 retro-active deaths over period before 30 March
```

Comment:
I suggest to move this one line lower and use it to explain why the `redistribute` is needed.